### DISCUSSION TITLE
Syntax of OIDCProviderRegistrationEndpointJson <json-string>

### DISCUSSION THREAD

**Discussion by silke** [OpenIDC/mod_auth_openidc#829] [Q&A] [answered]
Hi! :)
I am working on a setup with mod_auth_openidc and Shibboleth IdP (+ OIDC plugin). I can dynamically register my RPs with the OP. The RPs always get the sub claim configured as default on the OP side (subject-pairwise). Now I am looking for a way to make an RP ask for the subject-public claim.
If I understand correctly I can hand the subject_type over with the registration request and that should be this setting: `OIDCProviderRegistrationEndpointJson <json-string>`. Could you please help me get the **syntax** right? Whatever I try, Apache's config test doesn't like it. (I am not doing it in a .conf file but in the virtual host of a Relying Party.)
Greetings!

**Comment by zandbelt** [comment, marked as answer]
it should be something like:
```apache
OIDCProviderRegistrationEndpointJson "{ \"key1\": \"value1\", \"key2\": \"value2\" }"
```

**Reply by silke** [reply]
Thank you so much!
